youtrack_sdk/types.py

```diff
@@ -13,7 +13,7 @@
 ]
 
 
```

Comment:
Could you please define 


```python
YouTrackDateTime = Annotated[AwareDatetime, BeforeValidator(validate_youtrack_datetime)]
```

here and use it everywhere, where `AwareDatetime` is mixed with `str` instead of clamping the validator on the field at usage.